salt/etcd/init.sls

```diff
@@ -66,10 +66,15 @@ etcd:
     {%- if should_register_etcd_member %}
       - add-etcd-to-cluster
     {%- endif %}
-  # wait until etcd is actually up and running
+# wait until etcd is actually up and running
   caasp_etcd.healthy:
     - watch:
       - caasp_service: etcd
+    - retry:
+        attempts: 10
+        until: True
+        interval: 60
+        splay: 10
```

Comment:
We can probably just use `attempts` and `interval`. I think this interval is way too high though, what about `15` interval?

Reply:
okl i can try

Reply:
done